detox/test/e2e/p-datePicker.js

```diff
@@ -0,0 +1,17 @@
+describe('DatePicker', () => {
+    beforeEach(async () => {
+      await device.reloadReactNative();
+    });
+  
+    beforeEach(async () => {
+      await element(by.text('DatePicker')).tap();
+    });
+    
+    it('check and scroll datePicker', async () => {
+        await expect(element(by.type('UIPickerView'))).toBeVisible();
```

Comment:
Why by type and not by id? Does it not work?

Reply:
UIPickerView have not accessibility ID, testID = {'CustomDatePicker'} need to be deleted

Reply:
This is a problem. If user has more than one pickers, the API will break

Reply:
`UIPickerView` is a `UIView`, so it can have an accessibility ID. Are you seeing that `UIDatePicker` does not forward that ID?

Reply:
I checked that UI element via AcessebilityInspector (from Xcode Instruments), and now I found only one way to get element by type

Reply:
OK, thanks

Reply:
It won't work on android

Reply:
We know. We want to hone down iOS support first, then we will implement Android as well.